src/Message/InquireRequest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Omnipay\Cardconnect\Message;
+
+class InquireRequest extends AbstractRequest
+{
+    public function getData()
+    {
+        $this->validate('transactionReference');
+    	$data = array(
+    		'merchid'   => $this->getMerchantId(),
+    		'retref'   => $this->getTransactionReference()
+    	);
```

Comment:
If we are replacing `array()` with `[]` we should also do it here.
```suggestion
    	$data = [
    	    'merchid' => $this->getMerchantId(),
            'retref' => $this->getTransactionReference()
    	];
```

Reply:
copy pasta error